packages/cairo.rb

```diff
@@ -10,9 +10,11 @@ class Cairo < Package
   depends_on 'fontconfig'
   depends_on 'libpng'
   depends_on 'pixman'
+  depends_on 'freetype'   # pango requires cairo with freetype
```

Comment:
I modified this like below at https://github.com/skycocker/chromebrew/pull/1023#discussion_r132813714.  There is no order problem like you points.
```
  depends_on 'libpng'
  depends_on 'pixman'
  depends_on 'freetype'   # pango requires cairo with freetype
  depends_on 'fontconfig' # pango requires cairo with fontconfig
```